StatBot/StatBot/Config.cs

```diff
@@ -14,5 +14,8 @@ public class Config
 
         [JsonProperty("discord_token")]
         public string DiscordToken { get; set; }
+
+        [JsonProperty("command_prefix")]
+        public string CommandPrefix { get; set; }
```

Comment:
I believe technically, `CommandPrefix` should be labeled as a `char`

Reply:
It might make sense to include a very quick docstring on this, although it is rather self-explanatory imo, up2u